src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -94,10 +94,18 @@ define 'orifice-calculator-viewmodel', ['knockout', 'lodash', 'knockout.validati
       @velocityOfApproach = ko.computed =>
         _.ceil (1 / Math.sqrt(1 - @betaRatio() ** 4)), 2
 
+      @flowRateUnit = ko.observableArray _.values OPL.OrificeCalculator.Config.Dictionaries.FlowRateUnits
+      @selectedFlowRateUnit = ko.observable OPL.OrificeCalculator.Config.Dictionaries.FlowRateUnits.hour
```

Comment:
The default should be minute because its the one they our client uses more often